.xsession

```diff
@@ -3,3 +3,5 @@ setxkbmap -option caps:escape
 
 # Changes keyboard layout and modifier key
 setxkbmap -model pc105 -layout us,us -variant dvorak, -option grp:shifts_toggle,grp_led:caps
+
+statusbar_wrapper &
```

Comment:
I use dvorak as my keyboard layout and because of that I don't want other people to copy my .xsession. So can you move ```line 7``` to the .xinitrc file? (I have tried it out and it seems to be working fine)